notebooks/src/3-create-stac-item-with-extension.ipynb

```diff
@@ -297,22 +1575,157 @@
+       "   'common_name': 'nir09',\n",
+       "   'description': 'Near-IR2: 860 - 1040 nm'}],\n",
+       " 'platform': 'Maxar',\n",
+       " 'instruments': ['WorldView3'],\n",
+       " 'gsd': 0.3,\n",
+       " 'view:sun_azimuth': 160.86021018,\n",
+       " 'view:sun_elevation': 23.81656674}"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "item.properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Projection Extension\n",
+    "We are specifying the EPSG code for the image here manualy, but how to figures out the correct UTM Zone EPSG: follow this [tutorial](https://github.com/stac-utils/pystac/blob/fc839fb92b628f25eb1a54cfa0317c51429c5b9b/docs/tutorials/creating-a-landsat-stac.ipynb)\n",
+    "\n",
+    "![Projection Extension fields](../../assets/images/python-images/projection_extension.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proj_ext = ProjectionExtension.ext(item, add_if_missing=True)\n",
+    "proj_ext.epsg = 4326"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'datetime': '2023-10-16T09:08:53.705550Z',\n",
+       " 'eo:bands': [{'name': 'Coastal',\n",
+       "   'common_name': 'coastal',\n",
+       "   'description': 'Coastal: 400 - 450 nm'},\n",
+       "  {'name': 'Blue', 'common_name': 'blue', 'description': 'Blue: 450 - 510 nm'},\n",
+       "  {'name': 'Green',\n",
+       "   'common_name': 'green',\n",
+       "   'description': 'Green: 510 - 580 nm'},\n",
+       "  {'name': 'Yellow',\n",
+       "   'common_name': 'yellow',\n",
+       "   'description': 'Yellow: 585 - 625 nm'},\n",
+       "  {'name': 'Red', 'common_name': 'red', 'description': 'Red: 630 - 690 nm'},\n",
+       "  {'name': 'Red Edge',\n",
+       "   'common_name': 'rededge',\n",
+       "   'description': 'Red Edge: 705 - 745 nm'},\n",
+       "  {'name': 'Near-IR1',\n",
+       "   'common_name': 'nir08',\n",
+       "   'description': 'Near-IR1: 770 - 895 nm'},\n",
+       "  {'name': 'Near-IR2',\n",
+       "   'common_name': 'nir09',\n",
+       "   'description': 'Near-IR2: 860 - 1040 nm'}],\n",
+       " 'platform': 'Maxar',\n",
+       " 'instruments': ['WorldView3'],\n",
+       " 'gsd': 0.3,\n",
+       " 'view:sun_azimuth': 160.86021018,\n",
+       " 'view:sun_elevation': 23.81656674,\n",
+       " 'proj:epsg': 4326}"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "item.properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create a catalog, add the item that uses the EO, View and Projection extension, and save to a new STAC."
```

Comment:
Reword "Let's create a catalog, add the item that uses the EO, View and Projection extension, and save to a new STAC." to "Let's create a catalog, add the item that uses our extensions (EO, View, and Projection), and save to a new STAC."